tests/core/test_validation_config.py

```diff
@@ -491,3 +497,28 @@ def test_validation_config_deserialization_non_existant_resource(
     ):
         with pytest.raises(ValueError, match=f"{error_substring}*."):
             ValidationConfig.parse_obj(serialized_config)
+
+
+@pytest.mark.unit
+def test_validation_config_save_success(
+    ephemeral_context: EphemeralDataContext, validation_config: ValidationConfig
+):
+    context = ephemeral_context
+    vc = validation_config
+
+    vc = context.validations.add(vc)
```

Comment:
nit: to make the intention of the test here more clear, having the initial `context.validations.add(vc)` being done in a fixture might make sense, e.g. like a `saved_validation_config` fixture.